api/MediaDevices.json

```diff
@@ -112,15 +112,35 @@
             "firefox": [
               {
                 "version_added": "63",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "media.setsinkid.enabled",
+                    "value_to_set": "true"
+                  }
+                ],
+                "notes": "Prior to Firefox 63, <code>enumerateDevices()</code> only returned input devices. Starting with Firefox 63, output devices are also included."
+              },
+              {
+                "version_added": "39"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "63",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "media.setsinkid.enabled",
+                    "value_to_set": "true"
+                  }
+                ],
                 "notes": "Prior to Firefox 63, <code>enumerateDevices()</code> only returned input devices. Starting with Firefox 63, output devices are also included."
```

Comment:
Other than the note change mentioned above, which should be repeated here, these changes LGTM. With that text added, this will be ready to merge.